### PR TITLE
[PAY-1249] Display track and playlist tiles in web chats - Part 2

### PR DESCRIPTION
**Pull request by sddioulde**
### Description

Additional web chat track and playlist tile work that merges into https://github.com/AudiusProject/audius-client/pull/3377

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/web/src/components/track/mobile/PlaylistTile.tsx [hunk 1]
```diff
@@ -95,14 +95,14 @@ const TrackList = ({
           track={track}
         />
       ))}
-      {trackCount && trackCount > 5 && (
+      {(trackCount && trackCount > DISPLAY_TRACK_COUNT && trackCount > tracks.length) ? (
```
Comment (rickyrombo) [inline packages/web/src/components/track/mobile/PlaylistTile.tsx:98]: change logic below to subtract DISPLAY_TRACK_COUNT

File: packages/web/src/pages/chat-page/components/ChatMessageListItem.tsx [hunk 2]
```diff
@@ -124,7 +127,7 @@ export const ChatMessageListItem = (props: ChatMessageListItemProps) => {
       })}
     >
       <div className={styles.bubble}>
-        {!isTrackOrPlaylistLink
+        {!showTile
           ? links
```
Comment (rickyrombo) [inline packages/web/src/pages/chat-page/components/ChatMessageListItem.tsx:131]: I think we should use `links` and make the `map` into a switch/case or if/else sort of deal?

**Review by rickyrombo — APPROVED** [review]
amazzinngnngng

File: packages/web/src/pages/chat-page/components/ChatMessageListItem.tsx (hunk 2 above)
Comment (sddioulde) [reply packages/web/src/pages/chat-page/components/ChatMessageListItem.tsx]: yes!

**Pull request merged**